src/components/Calculator.jsx

```diff
@@ -8,6 +12,8 @@ import './Calculator.css';
 import React, { Component } from 'react';
 import ButtonWraper from './ButtonWraper';
 import Screen from './Screen';
+import Button from './Button';
+import calculate from '../logic/calculate';
 // import calculate from '../logic/calculate';
 // import operate from '../logic/operate';
 

```

Comment:
- [ ] <b>REQUIRED:</b> You overused `eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them.
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.